docs/source/user_guide/faq.rst

```diff
@@ -89,7 +93,8 @@ What frameworks does `fastplotlib` support?
         - `glfw`
         - `wxPython`
 
-    Note: Use in Google Colab does not currently work because Colab does not offer support for `jupyter_rfb`
+    Note: Use in Google Colab is not highly functional. We recommend using an inexpensive alternative cloud provider
+    such as CodeOcean or Lambda Cloud. Kushal has tested these and `fastplotlib` works very well.
```

Comment:
lmao I'm the scapegoat :joy:

Reply:
bahaha, I didn't mean it like that...fixed so it is less ambiguous who is to blame if it doesn't work lol